yt_dlp/extractor/radioradicale.py

```diff
@@ -0,0 +1,64 @@
+from locale import LC_ALL, setlocale
+from .common import InfoExtractor
+from urllib.parse import urlparse
+from re import sub
+from datetime import datetime
+from os.path import dirname
```

Comment:
```suggestion
import os.path
import re
import urllib.parse

from .common import InfoExtractor
```

* Sort imports
* Always import whole modules from stdlib